packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/utils/hasContent.js

```diff
@@ -0,0 +1,17 @@
+import isEmpty from 'lodash/isEmpty';
+
+export default function hasContent(type, content, metadatas, fieldSchema) {
+  if (type === 'component') {
+    const {
+      mainField: { name: mainFieldName },
+    } = metadatas;
+
+    if (fieldSchema?.repeatable) {
+      return content.some(item => !isEmpty(item));
```

Comment:
In this case you will always have an id field so you can use the same statement as below. Or maybe `content && content.length > 0`

Reply:
Fixed in 44921d68c2f3b94d626f91dc769aa255481dbd74